packages/tools/devtools/devtools-core/src/test/DataVisualizerGraph.test.ts

```diff
@@ -205,4 +206,29 @@ describe("DataVisualizerGraph unit tests", () => {
 		};
 		expect(childCellTree).to.deep.equal(expectedChildCellTree);
 	});
+
+	it("Unknown object in Container Data", async () => {
+		const unknownObject = {};
+
+		const visualizer = new DataVisualizerGraph(
+			{
+				unknownObject: unknownObject as IFluidLoadable,
+			},
+			defaultVisualizers,
+			defaultEditors,
+		);
+
+		const childUnknownObject = await visualizer.render("unknown-object");
```

Comment:
This should call `visualizer.renderRoot` (or whatever that method is called). The one that gives us all our root handles back. Then we can verify that it contains a single entry `unknownObject: unknownObjectNode`.

Reply:
Yes! I should have made the `childUnknownObject` to be `unknownObjectNode` as the `renderRootHandles()` would mark is as `unknownObjectNode`. 

I initially made a mistake of calling `render()` instead of `renderRootHandles()` to retrieve all the root handles. I have updated the logic.